.env

```diff
@@ -0,0 +1,13 @@
+# Django settings
+SECRET_KEY=dUmMy_s3cR3t_k3y
+DB_ENGINE=mysql
+DB_NAME=yaksh
+DB_USER=yaksh_user
+DB_PASSWORD=mysecretpassword
+#DB_HOST=yaksh-db
+DB_PORT=3306
+# Yaksh settings
+N_CODE_SERVERS=5
+SERVER_POOL_PORT=53579
+SERVER_HOST_NAME=http://yaksh-codeserver
+#SERVER_TIMEOUT=4
```

Comment:
Don't change  the `.sampleenv` to `.env`. This will lead to issue while setting up development environment, because mysql may not be used in the dev env.